packages/okta-adapter/src/config.ts

```diff
@@ -0,0 +1,712 @@
+      },
+      modify: {
+        url: '/api/v1/apps/{applicationId}',
+        method: 'put',
+        urlParamsToFields: {
+          applicationId: 'id',
+        },
+      },
+      remove: {
+        url: '/api/v1/apps/{applicationId}',
+        method: 'delete',
+        urlParamsToFields: {
+          applicationId: 'id',
+        },
+      },
+    },
+  },
+  AppUser: {
+    transformation: {
+      idFields: ['&id'],
+      extendsParentId: true,
+      fieldsToOmit: [
+        { fieldName: 'created' },
+        { fieldName: 'lastUpdated' },
+        { fieldName: 'statusChanged' },
+        { fieldName: '_links' },
+      ],
+    },
+    deployRequests: {
+      add: {
+        url: '/api/v1/apps/{applicationId}/users/{userId}',
+        urlParamsToFields: {
+          applicationId: '_parent.0.id',
+          userId: 'id',
+        },
+        method: 'post',
+        fieldsToIgnore: ['id', 'status', 'syncState'],
+      },
+      modify: {
+        url: '/api/v1/apps/{applicationId}/users/{userId}',
+        urlParamsToFields: {
+          applicationId: '_parent.0.id',
+          userId: 'id',
+        },
+        method: 'post',
+        fieldsToIgnore: ['id', 'status', 'syncState'],
+      },
+      remove: {
+        url: '/api/v1/apps/{applicationId}/users/{userId}',
+        urlParamsToFields: {
+          applicationId: '_parent.0.id',
+          userId: 'id',
+        },
+        method: 'delete',
+        fieldsToIgnore: ['id'],
+      },
+    },
+  },
+  'api__v1__apps___appId___credentials__keys@uuuuuu_00123_00125uuuu': {
+    transformation: {
+      dataField: '.',
+    },
+  },
+  api__v1__meta__types__user: {
+    transformation: {
+      // by default there is an unwanted traversal here
+      dataField: '.',
+    },
+  },
+  api__v1__users: {
+    request: {
+      url: '/api/v1/users',
+      recurseInto: [
+        {
+          type: 'api__v1__users___userId___roles@uuuuuu_00123_00125uu',
+          toField: 'roles',
+          context: [{ name: 'userId', fromField: 'id' }],
+        },
+      ],
+    },
+  },
+  api__v1__idps: {
+    request: {
+      url: '/api/v1/idps',
+      recurseInto: [
+        {
+          type: 'api__v1__idps___idpId___users@uuuuuu_00123_00125uu',
+          toField: 'users',
+          context: [{ name: 'idpId', fromField: 'id' }],
+        },
+        {
+          type: 'api__v1__idps___idpId___credentials__csrs@uuuuuu_00123_00125uuuu',
+          toField: 'CSRs',
+          context: [{ name: 'idpId', fromField: 'id' }],
+        },
+      ],
+    },
+  },
+  IdentityProvider: {
+    transformation: {
+      fieldTypeOverrides: [
+        { fieldName: 'users', fieldType: 'list<IdentityProviderApplicationUser>' },
+        { fieldName: 'CSRs', fieldType: 'list<Csr>' },
+      ],
+    },
+  },
+  api__v1__features: {
+    request: {
+      url: '/api/v1/features',
+      recurseInto: [
+        {
+          // Features that needs to be enabled in order to enable the feature
```

Comment:
nit:
```suggestion
          // Additional features that need to be enabled in order to enable the feature
```
(if I understood correctly)